<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatbotDialog</class>
 <widget class="QDialog" name="ChatbotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Management Chatbot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: #f8fafc;
    font-family: 'Segoe UI', Arial, sans-serif;
}

QTextEdit {
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 10px;
    background: white;
    font-size: 14px;
}

QLineEdit {
    padding: 10px 14px;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    font-size: 14px;
    background: white;
}

QLineEdit:focus {
    border-color: #667eea;
}

QPushButton {
    padding: 10px 16px;
    border-radius: 8px;
    font-weight: 500;
    background: #667eea;
    color: white;
    border: none;
}

QPushButton:hover {
    background: #5a67d8;
}

QListView {
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    background: white;
    font-size: 13px;
}

QListView::item {
    padding: 8px;
    border-bottom: 1px solid #f1f1f1;
}

QListView::item:hover {
    background: #edf2f7;
}

QListView::item:selected {
    background: #667eea;
    color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QTextEdit" name="chatHistory">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="suggestionLabel">
     <property name="text">
      <string>Suggestions:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
color: #4a5568;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="suggestionListView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="inputLineEdit">
       <property name="placeholderText">
        <string>Type your question here...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
